### PR TITLE
RealVNC.VNCServer version 6.11.0.47988

### PR DESCRIPTION
**Pull request by cooki371**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/85524)

### CODE REVIEW

**Comment by cooki371** [comment]
@microsoft-github-policy-service agree


Von: microsoft-github-policy-service[bot] ***@***.***>
Gesendet: Mittwoch, 19. Oktober 2022 22:33
An: microsoft/winget-pkgs ***@***.***>
Cc: Cooki ***@***.***>; Mention ***@***.***>
Betreff: Re: [microsoft/winget-pkgs] RealVNC.VNCServer version 6.11.0 (PR #85524)


@cooki371<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcooki371&data=05%7C01%7C%7C009110eea1a84efb010d08dab2112c8b%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638018084083898184%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=i5v%2BTSoXcRVYLw5Iq7Io1P5aEYenwWCtpQSfx7Xyh98%3D&reserved=0> please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.

@microsoft-github-policy-service agree [company="{your company}"]



Options:

  *   (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
  *   (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term "You" includes me and my employer.
Contributor License Agreement
Contribution License Agreement

This Contribution License Agreement ("Agreement") is agreed to by the party signing below ("You"),
and conveys certain license rights to Microsoft Corporation and its affiliates ("Microsoft") for Your
contributions to Microsoft open source projects. This Agreement is effective as of the latest signature
date below.

  1.  Definitions.
"Code" means the computer software code, whether in human-readable or machine-executable form,
that is delivered by You to Microsoft under this Agreement.
"Project" means any of the projects owned or managed by Microsoft and offered under a license
approved by the Open Source Initiative (www.opensource.org<https://nam12.safelinks.protection.outlook.com/?url=http%3A%2F%2Fwww.opensource.org%2F&data=05%7C01%7C%7C009110eea1a84efb010d08dab2112c8b%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638018084083898184%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=FhmpPOvZ%2BCB5COJTeP5r4HQBNs3C1tncPR2CJRAe%2Fjw%3D&reserved=0>).
"Submit" is the act of uploading, submitting, transmitting, or distributing code or other content to any
Project, including but not limited to communication on electronic mailing lists, source code control
systems, and issue tracking systems that are managed by, or on behalf of, the Project for the purpose of
discussing and improving that Project, but excluding communication that is conspicuously marked or
otherwise designated in writing by You as "Not a Submission."
"Submission" means the Code and any other copyrightable material Submitted by You, including any
associated comments and documentation.
  2.  Your Submission. You must agree to the terms of this Agreement before making a Submission to any
Project. This Agreement covers any and all Submissions that You, now or in the future (except as
described in Section 4 below), Submit to any Project.
  3.  Originality of Work. You represent that each of Your Submissions is entirely Your original work.
Should You wish to Submit materials that are not Your original work, You may Submit them separately
to the Project if You (a) retain all copyright and license information that was in the materials as You
received them, (b) in the description accompanying Your Submission, include the phrase "Submission
containing materials of a third party:" followed by the names of the third party and any licenses or other
restrictions of which You are aware, and (c) follow any other instructions in the Project's written
guidelines concerning Submissions.
  4.  Your Employer. References to "employer" in this Agreement include Your employer or anyone else
for whom You are acting in making Your Submission, e.g. as a contractor, vendor, or agent. If Your
Submission is made in the course of Your work for an employer or Your employer has intellectual
property rights in Your Submission by contract or applicable law, You must secure permission from Your
employer to make the Submission before signing this Agreement. In that case, the term "You" in this
Agreement will refer to You and the employer collectively. If You change employers in the future and
desire to Submit additional Submissions for the new employer, then You agree to sign a new Agreement
and secure permission from the new employer before Submitting those Submissions.
  5.  Licenses.

  *   Copyright License. You grant Microsoft, and those who receive the Submission directly or
indirectly from Microsoft, a perpetual, worldwide, non-exclusive, royalty-free, irrevocable license in the
Submission to reproduce, prepare derivative works of, publicly display, publicly perform, and distribute
the Submission and such derivative works, and to sublicense any or all of the foregoing rights to third
parties.
  *   Patent License. You grant Microsoft, and those who receive the Submission directly or
indirectly from Microsoft, a perpetual, worldwide, non-exclusive, royalty-free, irrevocable license under
Your patent claims that are necessarily infringed by the Submission or the combination of the
Submission with the Project to which it was Submitted to make, have made, use, offer to sell, sell and
import or otherwise dispose of the Submission alone or with the Project.
  *   Other Rights Reserved. Each party reserves all rights not expressly granted in this Agreement.
No additional licenses or rights whatsoever (including, without limitation, any implied licenses) are
granted by implication, exhaustion, estoppel or otherwise.

  1.  Representations and Warranties. You represent that You are legally entitled to grant the above
licenses. You represent that each of Your Submissions is entirely Your original work (except as You may
have disclosed under Section 3). You represent that You have secured permission from Your employer to
make the Submission in cases where Your Submission is made in the course of Your work for Your
employer or Your employer has intellectual property rights in Your Submission by contract or applicable
law. If You are signing this Agreement on behalf of Your employer, You represent and warrant that You
have the necessary authority to bind the listed employer to the obligations contained in this Agreement.
You are not expected to provide support for Your Submission, unless You choose to do so. UNLESS
REQUIRED BY APPLICABLE LAW OR AGREED TO IN WRITING, AND EXCEPT FOR THE WARRANTIES
EXPRESSLY STATED IN SECTIONS 3, 4, AND 6, THE SUBMISSION PROVIDED UNDER THIS AGREEMENT IS
PROVIDED WITHOUT WARRANTY OF ANY KIND, INCLUDING, BUT NOT LIMITED TO, ANY WARRANTY OF
NONINFRINGEMENT, MERCHANTABILITY, OR FITNESS FOR A PARTICULAR PURPOSE.
  2.  Notice to Microsoft. You agree to notify Microsoft in writing of any facts or circumstances of which
You later become aware that would make Your representations in this Agreement inaccurate in any
respect.
  3.  Information about Submissions. You agree that contributions to Projects and information about
contributions may be maintained indefinitely and disclosed publicly, including Your name and other
information that You submit with Your Submission.
  4.  Governing Law/Jurisdiction. This Agreement is governed by the laws of the State of Washington, and
the parties consent to exclusive jurisdiction and venue in the federal courts sitting in King County,
Washington, unless no federal subject matter jurisdiction exists, in which case the parties consent to
exclusive jurisdiction and venue in the Superior Court of King County, Washington. The parties waive all
defenses of lack of personal jurisdiction and forum non-conveniens.
  5.  Entire Agreement/Assignment. This Agreement is the entire agreement between the parties, and
supersedes any and all prior agreements, understandings or communications, written or oral, between
the parties relating to the subject matter hereof. This Agreement may be assigned by Microsoft.

-

File: manifests/r/RealVNC/VNCServer/6.11.0/RealVNC.VNCServer.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- silentWithProgress
+InstallerSwitches:
+  Silent: /qn REBOOT=ReallySuppress
+  SilentWithProgress: /qn REBOOT=ReallySuppress
+UpgradeBehavior: install
+AppsAndFeaturesEntries:
+- DisplayName: VNC Server 6.9.0
+  Publisher: RealVNC
+  ProductCode: '{63356231-70E1-46A7-98F3-1B00AF595756}'
+  InstallerType: msi
```
Comment (Trenly) [inline manifests/r/RealVNC/VNCServer/6.11.0/RealVNC.VNCServer.installer.yaml:19]:
```suggestion
AppsAndFeaturesEntries:
- DisplayName: VNC Server 6.11.0
  Publisher: RealVNC
  ProductCode: '{4D2F33DA-877A-4E1E-A292-614BBBCA7DBD}'
  InstallerType: msi
```

File: manifests/r/RealVNC/VNCServer/6.11.0/RealVNC.VNCServer.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ElevationRequirement: elevationRequired
+Installers:
+- Architecture: neutral
+  InstallerType: exe
+  InstallerUrl: https://downloads.realvnc.com/download/file/vnc.files/VNC-Server-6.11.0-Windows.exe
+  InstallerSha256: BBA55BCB47A0C85CCBD73F6EF8B826892EB2C769C6B80E17843DF61D188094B1
+  ProductCode: '{63356231-70E1-46A7-98F3-1B00AF595756}'
```
Comment (Trenly) [inline manifests/r/RealVNC/VNCServer/6.11.0/RealVNC.VNCServer.installer.yaml:26]:
```suggestion
  ProductCode: '{4D2F33DA-877A-4E1E-A292-614BBBCA7DBD}'
```

File: manifests/r/RealVNC/VNCServer/6.11.0/RealVNC.VNCServer.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: RealVNC.VNCServer
+PackageVersion: 6.11.0
```
Comment (Trenly) [inline manifests/r/RealVNC/VNCServer/6.11.0/RealVNC.VNCServer.installer.yaml:5]:
```suggestion
PackageVersion: 6.11.0.47988
```
Should match control panel

File: manifests/r/RealVNC/VNCServer/6.11.0/RealVNC.VNCServer.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: RealVNC.VNCServer
+PackageVersion: 6.11.0
```
Comment (Trenly) [inline manifests/r/RealVNC/VNCServer/6.11.0/RealVNC.VNCServer.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 6.11.0.47988
```
Should match control panel

File: manifests/r/RealVNC/VNCServer/6.11.0/RealVNC.VNCServer.yaml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: RealVNC.VNCServer
+PackageVersion: 6.11.0
```
Comment (Trenly) [inline manifests/r/RealVNC/VNCServer/6.11.0/RealVNC.VNCServer.yaml:5]:
```suggestion
PackageVersion: 6.11.0.47988
```
Should match control panel

**Review by Trenly — CHANGES_REQUESTED** [review]
The PackageVersion should match control panel; When updating this, you will need to update the name of the version folder also. You can do this in GitHub web by pressing <kbd>.</kbd>

**Review by stephengillie — APPROVED** [review]

**Pull request merged**